admin/tabs/userData/roles/recategorize/RecategorizeDialogModel.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2024 Extremely Heavy Industries Inc.
+ */
+import {RoleModel} from '@xh/hoist/admin/tabs/userData/roles/RoleModel';
+import {HoistModel, XH} from '@xh/hoist/core';
+import {Icon} from '@xh/hoist/icon/Icon';
+import {action, bindable, makeObservable, observable} from '@xh/hoist/mobx';
+import {compact, every, filter, map, uniq} from 'lodash';
+
+export class RecategorizeDialogModel extends HoistModel {
+    _parent: RoleModel;
+
+    @bindable categoryName = null;
+    @observable isOpen = false;
+
+    reCategorizeAction() {
+        return {
+            text: 'Change Category',
+            icon: Icon.folder(),
```

Comment:
I like the folder icon over `grip` here - but let's then use it for the dialog as well as adjust the use of the grip icon in configs/prefs "Change group" feature